src/core/runner.vala

```diff
@@ -0,0 +1,13 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private interface Games.Runner : Object {
+	public abstract Gtk.Widget? get_display ();
```

Comment:
> we have something to improve on

Definitely!

> Then use ZSNES and not GNOME Games : it's out of the application's focus.

I think embedding the game widget inside the app is also out of focus.
I wouldn't expect GNOME Documents to replace LibreOffice, for example.

Can you remind me what was the point in embedding the widget?
Was it for the snapshot/preview feature?